src/main/java/com/softserve/model/Poll.java

```diff
@@ -38,6 +35,6 @@ public class Poll {
     @Column(name = "modification_date")
     private Timestamp modificationDate;
 
-    @OneToMany(mappedBy = "poll")
+    @OneToMany(mappedBy = "poll", fetch = FetchType.EAGER)
```

Comment:
using EAGER is not a good practice, this can lead to memory leaks and overflows when having collections with other embedded collections.

The best approach is to read and fill the Entity values once those are really needed.

i.e.:   read list of "poll", then when interacting with an specific "poll", read its "pollOptions" using the corresponding service/repository,

Reply:
Ok I will fixed it, thnx